sceneview/src/main/java/com/google/ar/sceneform/ux/BaseGesture.java

```diff
@@ -108,7 +108,7 @@ public void onTouch(PickHitResult pickHitResult, MotionEvent motionEvent) {
 
   protected abstract void onFinish();
 
-  protected void cancel() {
+  public void cancel() {
```

Comment:
Needs to be public to access it from other package (io.github.sceneview). I could also move all Gesture related stuff to Kotlin and make it internal

Reply:
Coudl you include the Filament [https://github.com/google/filament/blob/main/android/filament-utils-android/src/main/java/com/google/android/filament/utils/GestureDetector.kt] ones
The Filament camera manipulator should also replace all those parts.